src/libs/auth.ts

```diff
@@ -428,9 +353,11 @@ export const isAzureUser = (): boolean => {
 };
 
 export const loadOidcUser = (user: OidcUser): void => {
-  return authStore.update((state) => {
+  oidcStore.update((state) => ({ ...state, user }));
+  authStore.update((state) => {
     const tokenClaims: B2cIdTokenClaims = user.profile;
     // according to IdTokenClaims, this is a mandatory claim and should always exist
+    // should be googleSubjectId or b2cId depending on how they authenticated
```

Comment:
This smells weird. Why are our IDs different depending on how they authenticate?

Reply:
This is just what id is send back from b2c's OAuth2, so this is whatever identifier that the google or microsoft uses (depending how you choose to auth). I do not believe we are using this id for anything in the UI